qiita_pet/static/js/networkVue.js

```diff
@@ -352,7 +352,7 @@ Vue.component('processing-graph', {
       })
         .fail(function(object, status, error_msg) {
           bootstrapAlert("Error submitting workflow: " + object.statusText, "danger");
-        });
```

Comment:
This is so the above and below statements belong together. If we leave there, we raise issues.